packages/@aws-cdk/aws-stepfunctions-tasks/test/databrew/integ.start-job-run.ts

```diff
@@ -0,0 +1,131 @@
+import * as databrew from '@aws-cdk/aws-databrew';
+import * as iam from '@aws-cdk/aws-iam';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import { DataBrewStartJobRun } from '../../lib';
+
+/*
+ * Stack verification steps:
+ * * aws stepfunctions start-execution --state-machine-arn <deployed state machine arn> : should return execution arn
+ * * aws stepfunctions describe-execution --execution-arn <exection-arn generated before> : should return status as SUCCEEDED
+ */
+
+class GlueDataBrewJobStack extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props: cdk.StackProps = {}) {
+    super(scope, id, props);
+
+    const region = process.env.CDK_DEPLOY_REGION || process.env.CDK_DEFAULT_REGION;
+
+    const outputBucket = new s3.Bucket(this, 'JobOutputBucket', {
+      removalPolicy: cdk.RemovalPolicy.DESTROY,
+    });
+
+    const role = new iam.CfnRole(this, 'DataBrew Role', {
```

Comment:
why `CfnRole` and not the L2 constructs?

Reply:
Changed.